ui/src/main/webapp/src/app/components/project-list/no-projects-welcome.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="blank-slate-pf">
-    <h1>
-        Welcome to the Red Hat Application Migration Toolkit Web Console
+    <h1 i18n>
+        Welcome to the Red Hat Application Migration Toolkit Web Console.
     </h1>
     <div class="blank-slate-pf-main-action">
-        <a (click)="newProjectClicked()" class="clickable btn btn-primary btn-lg">New Project</a>
+        <a (click)="newProjectClicked()" class="clickable btn btn-primary btn-lg" i18n="Button">New Project</a>
     </div>
 
-    <div class="welcome-help-text">
-        To learn more, visit the <a href="https://github.com/windup/windup-web/blob/master/README.md">
-            Red Hat Application Migration Toolkit Web Console Documentation</a>.
+    <div class="welcome-help-text"><template i18n="1st part">
```

Comment:
I mean that the tool creating the file could later include the file it came from, and perhaps some other context like nearest header and full css path, which could serve to the translator and lessen the need for explanatory descriptions.